### ISSUE TITLE
[Bug]: "New" button only shows "Nou" in French, instead of "Nouveau"

### ISSUE THREAD

**Issue by Dremor** [nextcloud/server#42390]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

In the "Files" app, the "New" button only shows the first 3 letters of "Nouveau" in French.
![image](https://github.com/nextcloud/server/assets/1915939/73d11d9f-320e-41cc-8cf4-5da5ef1011e0)

After closer inspection the CSS seem fine (it uses "width: fit-content", etc.), but only the three first letters are in the DOM span.
![image](https://github.com/nextcloud/server/assets/1915939/6d619654-f7ce-41a1-9972-21bcc6bbfb34)

### Steps to reproduce

1. Create a new NC 28 instance
2. Set language as French
3. Open Files app

### Expected behavior

The button should have shown "Nouveau"

### Installation method

Other Community project

### Nextcloud Server version

28

### Operating system

Other

### PHP engine version

PHP 8.2

### Web server

Nginx

### Database engine version

PostgreSQL

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "upgrade.disable-web": true,
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "28.0.0.11",
        "overwrite.cli.url": "https:\/\/cloud.canopus.dev",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "password": "***REMOVED SENSITIVE VALUE***",
            "port": "6379"
        },
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "default_phone_region": "FR",
        "share_folder": "Shared",
        "log_type": "file",
        "log_type_audit": "file",
        "loglevel": "2",
        "logfile": "\/var\/www\/html\/data\/logs\/nextcloud.log",
        "logfile_audit": "\/var\/www\/html\/data\/logs\/audit.log",
        "logdateformat": "d\/m\/Y H:i:s",
        "logtimezone": "Europe\/Paris",
        "overwritehost": "cloud.canopus.dev",
        "overwriteprotocol": "https",
        "trusted_domains": [
            "127.0.0.1",
            "localhost",
            "cloud-family-nextcloud",
            "cloud-family-nextcloud-*",
            "kube.internal.healthcheck",
            "cloud.canopus.dev",
            "192.168.0.10"
        ],
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "activity_expire_days": "90",
        "trashbin_retention_obligation": "auto",
        "versions_retention_obligation": "auto",
        "preview_imaginary_url": "http:\/\/cloud-family-nextcloud-imaginary:9090",
        "enabledPreviewProviders": [
            "OC\\Preview\\Imaginary",
            "OC\\Preview\\BMP",
            "OC\\Preview\\GIF",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\Krita",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\MP3",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\PNG",
            "OC\\Preview\\TXT",
            "OC\\Preview\\XBitmap"
        ],
        "enable_previews": "true",
        "jpeg_quality": "60",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "preview_max_memory": "1024",
        "preview_max_filesize_image": "50",
        "onlyoffice": ""
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.20.0
  - admin_audit: 1.18.0
  - bruteforcesettings: 2.8.0
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contacts: 5.5.0
  - contactsinteraction: 1.9.0
  - dashboard: 7.8.0
  - dav: 1.29.1
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - impersonate: 1.15.0
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - nextcloud_announcements: 1.17.0
  - notes: 4.9.1
  - notifications: 2.16.0
  - notify_push: 0.6.6
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - photos: 2.4.0
  - previewgenerator: 5.4.0
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recognize: 6.0.1
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.0
  - sharebymail: 1.18.0
  - support: 1.11.0
  - suspicious_login: 6.0.0
  - systemtags: 1.18.0
  - tasks: 0.15.0
  - text: 3.9.1
  - theming: 2.3.0
  - twofactor_backupcodes: 1.17.0
  - twofactor_totp: 10.0.0-beta.2
  - user_status: 1.8.1
  - viewer: 2.2.0
  - weather_status: 1.8.0
  - workflowengine: 2.10.0
Disabled:
  - encryption: 2.16.0
  - files_external: 1.20.0
  - survey_client: 1.16.0 (installed 1.16.0)
  - updatenotification: 1.18.0 (installed 1.18.0)
  - user_ldap: 1.19.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"EMl8S658zIkGe4JWp6xv","level":3,"time":"20/12/2023 09:53:41","remoteAddr":"192.168.0.254","user":"--","app":"index","method":"GET","url":"/apps/theming/favicon?v=b6589fc6","message":"Could not get appdata folder for theming","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0","version":"28.0.0.11","exception":{"Exception":"RuntimeException","Message":"Could not get appdata folder for theming","Code":0,"Trace":[{"file":"/var/www/html/lib/private/Files/AppData/AppData.php","line":147,"function":"getAppDataFolder","class":"OC\\Files\\AppData\\AppData","type":"->","args":[]},{"file":"/var/www/html/apps/theming/lib/ImageManager.php","line":367,"function":"newFolder","class":"OC\\Files\\AppData\\AppData","type":"->","args":["global"]},{"file":"/var/www/html/apps/theming/lib/ImageManager.php","line":101,"function":"getRootFolder","class":"OCA\\Theming\\ImageManager","type":"->","args":[]},{"file":"/var/www/html/apps/theming/lib/Controller/IconController.php","line":127,"function":"getImage","class":"OCA\\Theming\\ImageManager","type":"->","args":["favicon",false]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"getFavicon","class":"OCA\\Theming\\Controller\\IconController","type":"->","args":["core"]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Theming\\Controller\\IconController"],"getFavicon"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Theming\\Controller\\IconController"],"getFavicon"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Theming\\Controller\\IconController","getFavicon",["OC\\AppFramework\\DependencyInjection\\DIContainer"],["core","theming.Icon.getFavicon"]]},{"file":"/var/www/html/lib/base.php","line":1069,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/theming/favicon"]},{"file":"/var/www/html/index.php","line":39,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/lib/private/Files/AppData/AppData.php","Line":108,"message":"Could not get appdata folder for theming","exception":{},"CustomMessage":"Could not get appdata folder for theming"}}
{"reqId":"ZgUxPqotlrRgg8CvERxF","level":2,"time":"20/12/2023 09:53:58","remoteAddr":"192.168.0.254","user":"egeorget","app":"no app in context","method":"DELETE","url":"/remote.php/dav/files/egeorget/Reasons%20to%20use%20Nextcloud.pdf","message":"Retrying propagation query after retryable exception.","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0","version":"28.0.0.11","exception":{"Exception":"OC\\DB\\Exceptions\\DbalException","Message":"An exception occurred while executing a query: SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1119 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nProcess 1116 waits for ExclusiveLock on tuple (11,37) of relation 16444 of database 16385; blocked by process 1119.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":7,"Trace":[{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":328,"function":"wrap","class":"OC\\DB\\Exceptions\\DbalException","type":"::","args":[["Doctrine\\DBAL\\Exception\\DeadlockException"]]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":231,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437]},{"file":"/var/www/html/apps/files_trashbin/lib/Trashbin.php","line":322,"function":"renameFromStorage","class":"OC\\Files\\Cache\\Updater","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf","files_trashbin/files/Reasons to use Nextcloud.pdf.d1703062437"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/LegacyTrashBackend.php","line":108,"function":"move2trash","class":"OCA\\Files_Trashbin\\Trashbin","type":"::","args":["Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/TrashManager.php","line":102,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\LegacyTrashBackend","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":190,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\TrashManager","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":89,"function":"doDelete","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf","unlink"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1161,"function":"unlink","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Reasons to use Nextcloud.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/lib/private/DB/Exceptions/DbalException.php","Line":71,"Previous":{"Exception":"Doctrine\\DBAL\\Exception\\DeadlockException","Message":"An exception occurred while executing a query: SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1119 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nProcess 1116 waits for ExclusiveLock on tuple (11,37) of relation 16444 of database 16385; blocked by process 1119.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":7,"Trace":[{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1938,"function":"convert","class":"Doctrine\\DBAL\\Driver\\API\\PostgreSQL\\ExceptionConverter","type":"->","args":[["Doctrine\\DBAL\\Driver\\PDO\\Exception"],["Doctrine\\DBAL\\Query"]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1880,"function":"handleDriverException","class":"Doctrine\\DBAL\\Connection","type":"->","args":[["Doctrine\\DBAL\\Driver\\PDO\\Exception"],["Doctrine\\DBAL\\Query"]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1208,"function":"convertExceptionDuringQuery","class":"Doctrine\\DBAL\\Connection","type":"->","args":[["Doctrine\\DBAL\\Driver\\PDO\\Exception"],"UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba554e2f",1],[1,2,1]]},{"file":"/var/www/html/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba554e2f",1],[1,2,1]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", :dcValue1), \"etag\" = :dcValue3 WHERE (\"storage\" = :dcValue2) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,1,"6582aba554e2f"],[1,1,2]]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":231,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437]},{"file":"/var/www/html/apps/files_trashbin/lib/Trashbin.php","line":322,"function":"renameFromStorage","class":"OC\\Files\\Cache\\Updater","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf","files_trashbin/files/Reasons to use Nextcloud.pdf.d1703062437"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/LegacyTrashBackend.php","line":108,"function":"move2trash","class":"OCA\\Files_Trashbin\\Trashbin","type":"::","args":["Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/TrashManager.php","line":102,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\LegacyTrashBackend","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":190,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\TrashManager","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":89,"function":"doDelete","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf","unlink"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1161,"function":"unlink","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Reasons to use Nextcloud.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php","Line":35,"Previous":{"Exception":"Doctrine\\DBAL\\Driver\\PDO\\Exception","Message":"SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1119 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nProcess 1116 waits for ExclusiveLock on tuple (11,37) of relation 16444 of database 16385; blocked by process 1119.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":7,"Trace":[{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":132,"function":"new","class":"Doctrine\\DBAL\\Driver\\PDO\\Exception","type":"::","args":[["PDOException",["40P01",7,"ERROR:  deadlock detected\nDETAIL:  Process 1119 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nProcess 1116 waits for ExclusiveLock on tuple (11,37) of relation 16444 of database 16385; blocked by process 1119.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\""]]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1202,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba554e2f",1],[1,2,1]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", :dcValue1), \"etag\" = :dcValue3 WHERE (\"storage\" = :dcValue2) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,1,"6582aba554e2f"],[1,1,2]]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":231,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437]},{"file":"/var/www/html/apps/files_trashbin/lib/Trashbin.php","line":322,"function":"renameFromStorage","class":"OC\\Files\\Cache\\Updater","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf","files_trashbin/files/Reasons to use Nextcloud.pdf.d1703062437"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/LegacyTrashBackend.php","line":108,"function":"move2trash","class":"OCA\\Files_Trashbin\\Trashbin","type":"::","args":["Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/TrashManager.php","line":102,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\LegacyTrashBackend","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":190,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\TrashManager","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":89,"function":"doDelete","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf","unlink"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1161,"function":"unlink","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Reasons to use Nextcloud.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Exception.php","Line":28,"Previous":{"Exception":"PDOException","Message":"SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1119 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nProcess 1116 waits for ExclusiveLock on tuple (11,37) of relation 16444 of database 16385; blocked by process 1119.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":"40P01","Trace":[{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":130,"function":"execute","class":"PDOStatement","type":"->","args":[null]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1202,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba554e2f",1],[1,2,1]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", :dcValue1), \"etag\" = :dcValue3 WHERE (\"storage\" = :dcValue2) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,1,"6582aba554e2f"],[1,1,2]]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":231,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Reasons to use Nextcloud.pdf",1703062437]},{"file":"/var/www/html/apps/files_trashbin/lib/Trashbin.php","line":322,"function":"renameFromStorage","class":"OC\\Files\\Cache\\Updater","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf","files_trashbin/files/Reasons to use Nextcloud.pdf.d1703062437"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/LegacyTrashBackend.php","line":108,"function":"move2trash","class":"OCA\\Files_Trashbin\\Trashbin","type":"::","args":["Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Trash/TrashManager.php","line":102,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\LegacyTrashBackend","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":190,"function":"moveToTrash","class":"OCA\\Files_Trashbin\\Trash\\TrashManager","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/apps/files_trashbin/lib/Storage.php","line":89,"function":"doDelete","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf","unlink"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1161,"function":"unlink","class":"OCA\\Files_Trashbin\\Storage","type":"->","args":["files/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Reasons to use Nextcloud.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Reasons to use Nextcloud.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","Line":130}}},"message":"Retrying propagation query after retryable exception.","exception":{},"CustomMessage":"Retrying propagation query after retryable exception."}}
{"reqId":"12G0lOcNcjVSnT0vk5rY","level":2,"time":"20/12/2023 09:53:59","remoteAddr":"192.168.0.254","user":"egeorget","app":"no app in context","method":"DELETE","url":"/remote.php/dav/files/egeorget/Nextcloud%20Manual.pdf","message":"Retrying propagation query after retryable exception.","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0","version":"28.0.0.11","exception":{"Exception":"OC\\DB\\Exceptions\\DbalException","Message":"An exception occurred while executing a query: SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1116 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":7,"Trace":[{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":328,"function":"wrap","class":"OC\\DB\\Exceptions\\DbalException","type":"::","args":[["Doctrine\\DBAL\\Exception\\DeadlockException"]]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":170,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437]},{"file":"/var/www/html/lib/private/Files/View.php","line":301,"function":"remove","class":"OC\\Files\\Cache\\Updater","type":"->","args":["files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1173,"function":"removeUpdate","class":"OC\\Files\\View","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Nextcloud Manual.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/lib/private/DB/Exceptions/DbalException.php","Line":71,"Previous":{"Exception":"Doctrine\\DBAL\\Exception\\DeadlockException","Message":"An exception occurred while executing a query: SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1116 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":7,"Trace":[{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1938,"function":"convert","class":"Doctrine\\DBAL\\Driver\\API\\PostgreSQL\\ExceptionConverter","type":"->","args":[["Doctrine\\DBAL\\Driver\\PDO\\Exception"],["Doctrine\\DBAL\\Query"]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1880,"function":"handleDriverException","class":"Doctrine\\DBAL\\Connection","type":"->","args":[["Doctrine\\DBAL\\Driver\\PDO\\Exception"],["Doctrine\\DBAL\\Query"]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1208,"function":"convertExceptionDuringQuery","class":"Doctrine\\DBAL\\Connection","type":"->","args":[["Doctrine\\DBAL\\Driver\\PDO\\Exception"],"UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba556c35",1],[1,2,1]]},{"file":"/var/www/html/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba556c35",1],[1,2,1]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", :dcValue1), \"etag\" = :dcValue3 WHERE (\"storage\" = :dcValue2) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,1,"6582aba556c35"],[1,1,2]]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":170,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437]},{"file":"/var/www/html/lib/private/Files/View.php","line":301,"function":"remove","class":"OC\\Files\\Cache\\Updater","type":"->","args":["files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1173,"function":"removeUpdate","class":"OC\\Files\\View","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Nextcloud Manual.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php","Line":35,"Previous":{"Exception":"Doctrine\\DBAL\\Driver\\PDO\\Exception","Message":"SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1116 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":7,"Trace":[{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":132,"function":"new","class":"Doctrine\\DBAL\\Driver\\PDO\\Exception","type":"::","args":[["PDOException",["40P01",7,"ERROR:  deadlock detected\nDETAIL:  Process 1116 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\""]]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1202,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba556c35",1],[1,2,1]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", :dcValue1), \"etag\" = :dcValue3 WHERE (\"storage\" = :dcValue2) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,1,"6582aba556c35"],[1,1,2]]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":170,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437]},{"file":"/var/www/html/lib/private/Files/View.php","line":301,"function":"remove","class":"OC\\Files\\Cache\\Updater","type":"->","args":["files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1173,"function":"removeUpdate","class":"OC\\Files\\View","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Nextcloud Manual.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Exception.php","Line":28,"Previous":{"Exception":"PDOException","Message":"SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected\nDETAIL:  Process 1116 waits for ShareLock on transaction 2274; blocked by process 1117.\nProcess 1117 waits for ShareLock on transaction 2281; blocked by process 1116.\nHINT:  See server log for query details.\nCONTEXT:  while locking tuple (11,37) in relation \"oc_filecache\"","Code":"40P01","Trace":[{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":130,"function":"execute","class":"PDOStatement","type":"->","args":[null]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Connection.php","line":1202,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", ?), \"etag\" = ? WHERE (\"storage\" = ?) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,"6582aba556c35",1],[1,2,1]]},{"file":"/var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":386,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->","args":["UPDATE \"oc_filecache\" SET \"mtime\" = GREATEST(\"mtime\", :dcValue1), \"etag\" = :dcValue3 WHERE (\"storage\" = :dcValue2) AND (\"path_hash\" IN ('d41d8cd98f00b204e9800998ecf8427e', '45b963397aa40d4a0063e0d85e4fe7a1'))",[1703062437,1,"6582aba556c35"],[1,1,2]]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":280,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php","line":326,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/Propagator.php","line":137,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"/var/www/html/lib/private/Files/Cache/HomePropagator.php","line":48,"function":"propagateChange","class":"OC\\Files\\Cache\\Propagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437,0]},{"file":"/var/www/html/lib/private/Files/Cache/Updater.php","line":170,"function":"propagateChange","class":"OC\\Files\\Cache\\HomePropagator","type":"->","args":["files/Nextcloud Manual.pdf",1703062437]},{"file":"/var/www/html/lib/private/Files/View.php","line":301,"function":"remove","class":"OC\\Files\\Cache\\Updater","type":"->","args":["files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":1173,"function":"removeUpdate","class":"OC\\Files\\View","type":"->","args":[["OCA\\Files_Trashbin\\Storage",null,["OC\\Files\\Cache\\Scanner"],null,null,["OC\\Files\\Cache\\Updater"]],"files/Nextcloud Manual.pdf"]},{"file":"/var/www/html/lib/private/Files/View.php","line":702,"function":"basicOperation","class":"OC\\Files\\View","type":"->","args":["unlink","/Nextcloud Manual.pdf",["delete"]]},{"file":"/var/www/html/apps/dav/lib/Connector/Sabre/File.php","line":543,"function":"unlink","class":"OC\\Files\\View","type":"->","args":["/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php","line":185,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->","args":[]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->","args":["files/egeorget/Nextcloud Manual.pdf"]},{"file":"/var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["method:DELETE",[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[["Sabre\\HTTP\\Request"],["Sabre\\HTTP\\Response"]]},{"file":"/var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/lib/Server.php","line":370,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/html/remote.php","line":172,"args":["/var/www/html/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","Line":130}}},"message":"Retrying propagation query after retryable exception.","exception":{},"CustomMessage":"Retrying propagation query after retryable exception."}}
{"reqId":"4iRT5e9dIAAiniFf0VZk","level":3,"time":"20/12/2023 09:53:59","remoteAddr":"192.168.0.254","user":"egeorget","app":"index","method":"POST","url":"/apps/text/session/46/save","message":"No proper share data","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0","version":"28.0.0.11","exception":{"Exception":"InvalidArgumentException","Message":"No proper share data","Code":0,"Trace":[{"file":"/var/www/html/apps/text/lib/Service/ApiService.php","line":252,"function":"getFileForSession","class":"OCA\\Text\\Service\\DocumentService","type":"->","args":[["OCA\\Text\\Db\\Session",1],null]},{"file":"/var/www/html/apps/text/lib/Controller/SessionController.php","line":100,"function":"save","class":"OCA\\Text\\Service\\ApiService","type":"->","args":[["OCA\\Text\\Db\\Session",1],["OCA\\Text\\Db\\Document",46],0,"# Welcome to Nextcloud! \ud83d\udcf1 \u2601\ufe0f \ud83d\udcbb\n\nHere you can add a description or any other info relevant for the folder. It will show as a \"Readme.md\" and in the web interface also embedded nicely up at the top.","AQyFocX3BgAHAQdkZWZhdWx0AwdoZWFkaW5nBwCFocX3BgAGBACFocX3BgEmV2VsY29tZSB0byBOZXh0Y2xvdWQhIPCfk7Eg4piB77iPIPCfkrsoAIWhxfcGAAVsZXZlbAF9ASgAhaHF9wYAAmlkAX8oAIWhxfcGAAR1dWlkAX+HhaHF9wYAAwlwYXJhZ3JhcGgHAIWhxfcGIwYEAIWhxfcGJKQBSGVyZSB5b3UgY2FuIGFkZCBhIGRlc2NyaXB0aW9uIG9yIGFueSBvdGhlciBpbmZvIHJlbGV2YW50IGZvciB0aGUgZm9sZGVyLiBJdCB3aWxsIHNob3cgYXMgYSAiUmVhZG1lLm1kIiBhbmQgaW4gdGhlIHdlYiBpbnRlcmZhY2UgYWxzbyBlbWJlZGRlZCBuaWNlbHkgdXAgYXQgdGhlIHRvcC6HhaHF9wYjAwlwYXJhZ3JhcGiohaHF9wYhAXcWaC13ZWxjb21lLXRvLW5leHRjbG91ZKiFocX3BiIBdyQ3NGQzNDBmNi1jOWZmLTRjYWMtOGQ1Zi03ODdiN2EwMDI3ODgBhaHF9wYBIQI=",false,true]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"save","class":"OCA\\Text\\Controller\\SessionController","type":"->","args":[0,"# Welcome to Nextcloud! \ud83d\udcf1 \u2601\ufe0f \ud83d\udcbb\n\nHere you can add a description or any other info relevant for the folder. It will show as a \"Readme.md\" and in the web interface also embedded nicely up at the top.","AQyFocX3BgAHAQdkZWZhdWx0AwdoZWFkaW5nBwCFocX3BgAGBACFocX3BgEmV2VsY29tZSB0byBOZXh0Y2xvdWQhIPCfk7Eg4piB77iPIPCfkrsoAIWhxfcGAAVsZXZlbAF9ASgAhaHF9wYAAmlkAX8oAIWhxfcGAAR1dWlkAX+HhaHF9wYAAwlwYXJhZ3JhcGgHAIWhxfcGIwYEAIWhxfcGJKQBSGVyZSB5b3UgY2FuIGFkZCBhIGRlc2NyaXB0aW9uIG9yIGFueSBvdGhlciBpbmZvIHJlbGV2YW50IGZvciB0aGUgZm9sZGVyLiBJdCB3aWxsIHNob3cgYXMgYSAiUmVhZG1lLm1kIiBhbmQgaW4gdGhlIHdlYiBpbnRlcmZhY2UgYWxzbyBlbWJlZGRlZCBuaWNlbHkgdXAgYXQgdGhlIHRvcC6HhaHF9wYjAwlwYXJhZ3JhcGiohaHF9wYhAXcWaC13ZWxjb21lLXRvLW5leHRjbG91ZKiFocX3BiIBdyQ3NGQzNDBmNi1jOWZmLTRjYWMtOGQ1Zi03ODdiN2EwMDI3ODgBhaHF9wYBIQI=",false,true]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Text\\Controller\\SessionController"],"save"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Text\\Controller\\SessionController"],"save"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Text\\Controller\\SessionController","save",["OC\\AppFramework\\DependencyInjection\\DIContainer"],["46","text.Session.save"]]},{"file":"/var/www/html/lib/base.php","line":1069,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/text/session/46/save"]},{"file":"/var/www/html/index.php","line":39,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/apps/text/lib/Service/DocumentService.php","Line":447,"message":"No proper share data","exception":{},"CustomMessage":"No proper share data"}}
{"reqId":"0Y4Cf5cHNKb438RlfwJP","level":3,"time":"20/12/2023 09:53:59","remoteAddr":"192.168.0.254","user":"egeorget","app":"index","method":"POST","url":"/apps/text/session/46/sync","message":"No proper share data","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0","version":"28.0.0.11","exception":{"Exception":"InvalidArgumentException","Message":"No proper share data","Code":0,"Trace":[{"file":"/var/www/html/apps/text/lib/Service/ApiService.php","line":226,"function":"getFileForSession","class":"OCA\\Text\\Service\\DocumentService","type":"->","args":["*** sensitive parameters replaced ***",null]},{"file":"/var/www/html/apps/text/lib/Controller/SessionController.php","line":88,"function":"sync","class":"OCA\\Text\\Service\\ApiService","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"sync","class":"OCA\\Text\\Controller\\SessionController","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Text\\Controller\\SessionController"],"sync"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Text\\Controller\\SessionController"],"sync"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Text\\Controller\\SessionController","sync",["OC\\AppFramework\\DependencyInjection\\DIContainer"],["46","text.Session.sync"]]},{"file":"/var/www/html/lib/base.php","line":1069,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/text/session/46/sync"]},{"file":"/var/www/html/index.php","line":39,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/apps/text/lib/Service/DocumentService.php","Line":447,"message":"No proper share data","exception":{},"CustomMessage":"No proper share data"}}
```

### Additional info

Truenas Scale 23.10.1
Truecharts Nextcloud Chart

**Comment by joshtrichards** [comment]
Duplicate of #42372
Fixed in upcoming v28.0.1
